public/app/features/alerting/unified/components/rule-editor/QueryRows.tsx

```diff
@@ -169,6 +169,11 @@ export class QueryRows extends PureComponent<Props> {
                       error = errorFromPreviewData(data);
                     }
 
+                    if (error) {
+                      // TODO: use error correctly
```

Comment:
Let's fix this somehow :) at least by adding some logging maybe?

Reply:
Hm I remember this error propagating to the `ExtrasHeader` – can we investigate why this is no longer used? This _feels_ like a regression.

Reply:
Ok so according to `git bisect` this was changed in https://github.com/grafana/grafana/pull/80078 and we're now showing those errors somewhere else entirely – so feel free to clean this up :)